components/common/btn/selectButtons.tsx

```diff
@@ -1,26 +1,24 @@
 import styled from '@emotion/styled';
 
-interface SelectBtnProps {
+interface InSelectButtonpropsType {
   children: React.ReactNode;
 }
 
-const Select = styled.button<SelectBtnProps>`
+export default function SelectButtons({ children }: InSelectButtonpropsType) {
+  return <Select>{children}</Select>;
+}
+
+const Select = styled.button<InSelectButtonpropsType>`
   display: flex;
   align-items: center;
   justify-content: center;
 
   height: 2.5rem;
   padding: 0.7rem 1.1rem;
 
-  font-size: 1rem;
+  background-color: ${({ theme }) => theme.color.white};
+  border: 1px solid var(--black, ${({ theme }) => theme.color.black});
```

Comment:
```suggestion
  border: 1px solid ${({ theme }) => theme.color.black};
```
바로 요렇게 써도 될텐데.......?!??!

Reply:
엇 그러게 바로 수정 나이거 픽마에서 가져와서 전혀 모르고 지나쳤네!!